test/lib/render/CMakeLists.txt

```diff
@@ -0,0 +1 @@
+add_subdirectory(pxrUsdMayaGL)
```

Comment:
@mattyjams The folder structure under test should match the folder structure of the source tree. I know we have currently some cleanup to do with ufe, but can you adjust this folder structure? So instead of `test/lib/render/` be `test/lib/mayaUsd/render` ?